proposals/0035-linalg-matrix.md

```diff
@@ -1632,14 +1721,33 @@ ByteAddressBuffer MBuf : register(t0);
 
 void CoopVec() {
   using namespace dx::linalg;
-  using MatrixBTy = Matrix<MatrixComponentType::F16, 32, 16, MatrixUse::B,
+  using MatrixBTy = Matrix<MatrixComponentType::F16, 16, 16, MatrixUse::B,
                            MatrixScope::Thread>;
 
-  vector<float16_t, 32> Vec = (vector<float16_t, 32>)0;
+  vector<float16_t, 16> Vec = (vector<float16_t, 16>)0;
   MatrixBTy MatB = MatrixBTy::Load(
       MBuf, 0, /* Row stride = number of columns * element size */ 16 * 4,
       MatrixLayout::RowMajor);
-  vector<float16_t, 16> Accum = Multiply<float16_t>(Vec, MatB);
+  vector<float16_t, 16> Layer1 = Multiply<float16_t>(Vec, MatB);
+
+  vector<float16_t, 16> NullBias = (vector<float16_t, 16>)0;
+  vector<float16_t, 16> Layer2 = MultiplyAdd<float16_t>(Layer1, MatB, NullBias);
+
+  VectorRef<MatrixComponentType::F8_E4M3, 16> MemBias = {MBuf,
+                                                         /*start offset*/ 4096};
+  vector<float16_t, 16> Layer3 = MultiplyAdd<float16_t>(Layer2, MatB, MemBias);
+
+  // Clang doesn't yet support packed types.
+#ifdef __hlsl_dx_compiler
+  vector<uint8_t4_packed, 4> SomeData = (vector<uint8_t4_packed, 4>)0;
+
+  vector<float16_t, 16> Layer4 = MultiplyAdd<float16_t>(
+      MakeInterpretedVector<MatrixComponentType::F8_E4M3>(SomeData), MatB,
```

Comment:
Do we want to support Packed->Float conversion of input vectors? We don't support this currently.